qemu/tests/cfg/blockdev_mirror_qemuio_ready_job.cfg

```diff
@@ -22,5 +22,5 @@
     image_name_mirror1 = mirror1
 
     storage_type_default = directory
-    qemuio_cmd = '%s -f ${image_format_data1} %s'
+    qemuio_cmd = '%s -f %s %s'
```

Comment:
Seems the format is hard coded as qcow2, so we need to delete following lines.

 19     image_format_data1 = qcow2
 20     image_format_mirror1 = qcow2

Reply:
if we set 'image_format_mirror1 = raw', that would skip the qcow2 format mirror1 test, right? what's the test matrix here?
data1: raw/qcow2
mirror1: raw/qcow2
there should be 4 combinations

Reply:
Hi, zhenchao,
   the matrix here is: raw->raw, qcow2->raw, we can cover the left ones in other cases. In other cases, format for mirror1 is qcow2.

Reply:
Got it, then it might be better to add comments in cfg to clarify the test matrix, i.e. why we use a 'raw' mirror image here